include/inexor/vulkan-renderer/wrapper/command_buffer.hpp

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include <spdlog/spdlog.h>
+#include <vulkan/vulkan_core.h>
+
+namespace inexor::vulkan_renderer::wrapper {
+class CommandBuffer {
+private:
+    VkCommandBuffer command_buffer;
+
+public:
+    /// Delete the copy constructor so command buffers are move-only objects.
+    CommandBuffer(const CommandBuffer &) = delete;
+    CommandBuffer(CommandBuffer &&other) noexcept;
+
+    /// Delete the copy assignment operator so command buffers are move-only objects.
+    CommandBuffer &operator=(const CommandBuffer &) = delete;
+    CommandBuffer &operator=(CommandBuffer &&other) noexcept = default;
+
+    CommandBuffer(const VkDevice device, const VkCommandPool command_pool);
+
+    /// @note We don't need to destroy the command buffer because it will be destroyed along with it's associated command pool automatically.
+    ~CommandBuffer() = default;
+
+    [[nodiscard]] VkCommandBuffer get() const {
+        return command_buffer;
+    }
+
+    [[nodiscard]] auto get_ptr() const {
```

Comment:
We should be explicit and replace `auto` with `VkCommandBuffer *`

Reply:
Done.